instructions/bundling-scripts/zimbra-store.sh

```diff
@@ -155,9 +155,6 @@ main()
       mkdir -p ${repoDir}/zm-build/${currentPackage}/opt/zimbra/jetty_base/webapps/zimbra/public
     fi
 
-    echo "\t\t***** help content *****" >> ${buildLogFile}
```

Comment:
we should remove entries which clones zm-help repo and does some preprocessing before this
https://github.com/Zimbra/zm-build/blob/develop/instructions/FOSS_repo_list.pl#L26
https://github.com/Zimbra/zm-build/blob/develop/instructions/FOSS_staging_list.pl#L174

Reply:
Done, removed the above references of zm-help from code.